bitmap-weasels/lib/read-write.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+const readWrite = module.exports = {};
+
+const tranf = require('./transform');
+const Bitmap = require('./bitmap');
+const fs = require('fs');
+
+readWrite.bmp;
+readWrite.nameKeyThing;
+
+readWrite.read = () => {
+  fs.readFile(`${__dirname}/../assets/palette-bitmap.bmp`, (err, data) => {
+    if(err) console.error(err);
+    readWrite.bmp = new Bitmap(data);
+    readWrite.black();
+    fs.readFile(`${__dirname}/../assets/palette-bitmap.bmp`, (err, data) => {
+      if(err) console.error(err);
+      readWrite.bmp = new Bitmap(data);
+      readWrite.gray();
+      fs.readFile(`${__dirname}/../assets/palette-bitmap.bmp`, (err, data) => {
+        if(err) console.error(err);
+        readWrite.bmp = new Bitmap(data);
+        readWrite.invert();
+      });
+    });
+  });
+};
+
+readWrite.write = (fileContent, filePath) => {
+  fileContent = readWrite.bmp.allData;
+  filePath = `./assets/${readWrite.nameKeyThing}.bmp`;
+  fs.writeFile(filePath, fileContent, (err) => {
+    if (err) throw err;
+    console.log(`Created a Bitmap!`);
+  });
+};
+
+readWrite.black = () => {
```

Comment:
These wrappers are unnecessary given the refactor that I talked about in the last comment.